sag_et.par

```diff
@@ -15,88 +19,127 @@ n 14 8
 n 15 9
 /|  N\  /N\
 n 16 9
-)N(  M)
+|)  (|\
 d 17 10
 /|\  /N\
 d 19 11
 /N\
+n 20 12
+)~|  (|\  ~N\  /N\

```

Comment:
I've never seen this N = || M = ||| U = !! W = !!! notation anywhere except in this configuration file. I experimented with replacing N's in this file with ||, but then that caused things in Scala to show up as N's! Quite confusing. So I've reverted back to N's.

Reply:
Yeah. It just made it easier for Manuel to write code to parse it in Scala. We wouldn't want to use it anywhere else. These .par files are not really meant to be read by humans. You have to use the N's.

Reply:
I'll change back to the N's.

Reply:
Thanks.